### DISCUSSION TITLE
As the host of a multiplayer lobby the remove player overlaps with each players "have map/dont have map" status

### DISCUSSION THREAD

**Discussion by Will-Quinn-1919** [ppy/osu#22758] [Ideas]
Will quickly summarize:

As the host of a multiplayer lobby the remove player overlaps with each players "have map/dont have map" status and makes it hard to read

**Comment by mk56-spn** [comment]
videos would be appreciated given they are visual bugs. and please don't post bugs together in the same discussion, it makes working through them a hassle

**Comment by Theighlin** [comment]
First point is https://github.com/ppy/osu-web/issues/9689
